jekyll/_cci1/test-metadata.md

```diff
@@ -207,6 +208,36 @@ See the [minitest-ci README](https://github.com/circleci/minitest-ci#readme) for
 #### <a name="test2junit-for-clojure-tests"></a>test2junit for Clojure tests
 You can use [test2junit](https://github.com/ruedigergad/test2junit) to convert Clojure test output to XML format. For more details, please checkout our [sample project](https://github.com/kimh/circleci-build-recipies/tree/clojure-test-metadata-with-test2junit).
 
+#### <a name="karma"></a>Karma
+
+To output JUnit tests with the Karma test runner you can use [karma-junit-reporter](https://www.npmjs.com/package/karma-junit-reporter)
+
+A working [circle.yml]( {{ site.baseurl }}/1.0/configuration/) section for testing might look like this:
```

Comment:
Edit:

A working [circle.yml]( {{site.baseurl }}/1.0/configuration/) `test` section might look like this: